pydicom/dataelem.py

```diff
@@ -29,42 +28,11 @@
     PersonName = PersonNameUnicode
 
 
-# Helper functions:
-def isMultiValue(value):
-    """Return True if `value` is list-like (iterable),
```

Comment:
So... I'm still in favor of deprecating `isMultiValue` before completely removing it, in case someone has used it in user code (as the OP of the issue was doing).  Or at least give an error message with alternate instructions.  I think the number of uses must be very small, so I'm willing to forgo this, but it seems like a small cost (at least the second option certainly is).  I suppose the same could be said for `isStringOrStringList`, but it is more recent, and less likely to have been used.

Reply:
Ok, I will get it `isMultiValue` back in with a deprecation warning, but not use it anywhere. I'll leave `isStringOrStringList` and `isString` out and add a comment in the release notes then, if that would be ok.

Reply:
That sounds perfect.  Thanks.

Reply:
Ok, done.